ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -72,3 +80,10 @@ tests:
     set' run-tests
   openshift_installer:
     cluster_profile: openstack
+- as: e2e-libvirt
+  commands: TEST_FOCUS='Secrets should be consumable from pods in volume with defaultMode
+    set' run-tests
+  openshift_installer:
+    # TODO: NEED A secret with 1)gce.json 2)pull-secret for installing a libvirt cluster in a gce instance
+    #cluster_profile: libvirt-gcp
+    cluster_profile: gcp
```

Comment:
@stevekuznetsov for this job we'll need a secret with 1) gce.json  and 2) quay pull secret -already used in e2e-aws 
Also, I've been testing with GOOGLE_PROJECT_ID `openshift-gce-devel` but we'll want this to launch an instance in `openshift-gce-devel-ci`, right?  This is working well from the template (haven't tested the presubmit yaml).